fuse/opcode.go

```diff
@@ -450,7 +450,21 @@ func doStatFs(server *Server, req *request) {
 }
 
 func doIoctl(server *Server, req *request) {
-	req.status = ENOSYS
+	in := (*IoctlIn)(req.inData)
+	out := (*IoctlOut)(req.outData())
+	var bufIn, bufOut []byte
+	if in.InSize > 0 {
+		bufIn = req.arg
+	} else if in.OutSize > 0 {
```

Comment:
Is it possible that in.OutSize > 0 when in.InSize > 0?  We don't need to make this assumption.